<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuardDetailInfoDialog</class>
 <widget class="QDialog" name="GuardDetailInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>详细信息</string>
  </property>
  <widget class="QLabel" name="labelPic">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>80</width>
     <height>112</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
     border-width: 1px;
     border-style: dotted;
     border-color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>照片</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWorkPosition">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelIDCardNum">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>114</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>姓    名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>工    号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>工作地点</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>身份证号</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>61</y>
     <width>62</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="labelName">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelID">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>GuardDetailInfoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
